scripts/auth-common.sh

```diff
@@ -15,7 +13,38 @@ else
 fi
 
 IRONIC_HTPASSWD_FILE=/etc/ironic/htpasswd
+if [[ -f "/auth/ironic/username" ]]; then
+    read -r IRONIC_HTPASSWD_USERNAME<"/auth/ironic/username"
+fi
+IRONIC_HTPASSWD_USERNAME=${IRONIC_HTPASSWD_USERNAME:-}
+if [[ -f "/auth/ironic/password" ]]; then
+    read -r IRONIC_HTPASSWD_PASSWORD<"/auth/ironic/password"
+fi
+IRONIC_HTPASSWD_PASSWORD=${IRONIC_HTPASSWD_PASSWORD:-}
+if [[ -n "${IRONIC_HTPASSWD_USERNAME}" ]]; then
```

Comment:
I wonder if it's going to complain about undefined variables. Maybe you need "${IRONIC_HTPASSWD_USERNAME:-}"

Reply:
Its there on line 19 :)